posts/2024-07-24_working__groups_.../working__groups__updates.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Working Groups Updates"
+author:
+  - name: pharmaverse council
+description: "An update from pharmaverse council around how packages will get included in pharmaverse in future."
+date: "2024-07-24"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Community]
+# feel free to change the image
+image: "pharmaverse.PNG"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-07-24_working__groups_..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Hi community,
+
+We wanted to share with you an update from council discussions around our
+pharmaverse working groups. Up until now, these have been the decision-holders of
+which packages are included/excluded from the pharmaverse. Although open for
+all to join, they ended up being quite lean and likely not representative of the
+true depth of this community. So we’ve decided to open up the package decisions
+to our full community in future via our [Slack workspace](https://pharmaverse.slack.com/), to make for a more
+inclusive community where everyone has a voice. This is now updated on our
```

Comment:
Does this mean that this is now a democratic process, i.e. with voting on the packages or something? Or how exactly will the community decide to include the packages under the pharmaverse?

Reply:
I didn't want to make it overly prescriptive and rules-based as we should be able to evolve this as we learn from how it goes, but general thinking:

- we share a new package request to pharmaverse slack and ask community to either thumbs up on GH issue, or add a comment if any concerns
- if any concerns tag the requestor so they get a chance to respond and potentially allay the concern
- if any comment is not resolved and even just 1 member of our community still has an issue with a package being included then we'll take it to the council to adjudicate and a transparent decision with rationale will be added to the GH issue

So everyone has a voice... even 1 single member could stop a package being included if they have a good enough justification! If we don't hear any concerns (and the package fits scope of pharmaverse and inclusion criteria) then we'll go ahead and add it. 

If at any time in future a member of the community raises a concern then we could re-visit and see if any package should be removed.

I'll add a little bit more to the blog but don't want to go over the top either with explaining this.

FYI @bms63

Reply:
Would you mind adding this to the blog as well? 
Or maybe hint towards this, if it is not quite clear yet?

Reply:
@StefanThoma which bit exactly?

sorry, i added the following as a more concise summary of the above but happy to add some extra lines for whichever bits missed which you feel i should add

```
We don't want to be overly prescriptive with how this process will work as it
might evolve over time, but we'll be openly sharing any future package
applications to pharmaverse via our Slack and we welcome your input to help make
the call on which to accept/decline. Any individual from our community could
express support via a thumbs up on the GitHub issue or add a comment with any
concerns. If the requestor is unable to resolve any concerns, then ultimately
any contentious applications will be raised to the pharmaverse council to
adjudicate. The final decision and rationale will then always be documented on
the issue.
```